src/Application/Service/Property/__snapshots__/PropertyViewRequest.spec.js.snap

```diff
@@ -0,0 +1,13 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Do you think we can get of rid of this if we use typescript?

Reply:
No, but we can use the specific message instead of using snapshots

Reply:
I would avoid using folders name starting by __ and if this is part of tests I would differentiate source code from tests by moving them into tests folder